HW4_Diana_Mokhnitskaya/HW4_task3.py

```diff
@@ -0,0 +1,17 @@
+friends_list = ["John", "Marta", "James"]
+enemies_list = ["John", "Johnatan", "Artur"]
+for friend in friends_list:
+    if friend == "James":
+        break
```

Comment:
why break?
what if James will be as first element of the list?

Reply:
Fixed it.